Include/internal/pycore_interp.h

```diff
@@ -321,6 +322,7 @@ struct _is {
 
     struct ast_state ast;
     struct type_cache type_cache;
+    Bigint *bigint_freelist[_PyDtoa_Kmax+1];
```

Comment:
You need to add:

```
#ifndef PY_NO_SHORT_FLOAT_REPR
...
#endif 
```

If the PY_NO_SHORT_FLOAT_REPR macro is defined, dtoa.c is not used.